### PR TITLE
Fix failing media player caption tests

### PR DESCRIPTION
**Pull request by jamesdonoh**
Fix tests that were added in https://github.com/bbc/simorgh/pull/4256 that were failing when smoke testing during live deploy because the media player is not enabled in the live environment.

```
articles - news - Canonical Running tests for news articles Article Body should have a visible caption beneath a mediaplayer:
CypressError: Timed out retrying: Expected to find element: '1', but never found it. Queried from element: <figcaption.Caption-sc-16x70so-0.gDfBZX>
```

Fix verified locally and against test and live.

**Code changes:**

- Make tests dependent on mediaplayer toggle state
- Reorder for consistency

---

- [x] I have assigned myself to this PR and the corresponding issues
- [NA] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [x] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [no] This PR requires manual testing

### CODE REVIEW

**Review by jamesbrumpton — APPROVED** [review]

**Review by tochwill — APPROVED** [review]

**Review by 12 — APPROVED** [review]
Ah this was my bad. Thanks for fixing 👍

**Review by simonsinclair — APPROVED** [review]
`appToggles` to the rescue.

File: cypress/integration/pages/articles/tests.js [hunk 1]
```diff
@@ -213,6 +190,35 @@ export const testsThatFollowSmokeTestConfig = ({
           );
         });
       }
+
+      // `appToggles` tells us whether a feature is toggled on or off in the current environment.
+      if (appToggles.mediaPlayer.enabled) {
+        describe('Media Player', () => {
+          it('should have a visible caption beneath a mediaplayer', () => {
```
Comment (simonsinclair) [inline cypress/integration/pages/articles/tests.js:197]: This has just failed on CI:
```
Running tests for persian articles
      Metadata
        ✓ should have the correct articles metadata
        ✓ should include mainEntityOfPage in the LinkedData
      Article Body
        ✓ should render a H1, which contains/displays a styled headline (61ms)
        ✓ should render an H2, which contains/displays a styled subheading (56ms)
        ✓ should render a paragraph, which contains/displays styled text (87ms)
        ✓ should have a placeholder image (52ms)
        ✓ should have an image copyright label with styling (198ms)
        ✓ should have an inline link (54ms)
        Media Player
          1) should have a visible caption beneath a mediaplayer
```
I've seen this before for `persian` and believe it's our 'kitchen sink' problem. A check similar to [this one](https://github.com/bbc/simorgh/blob/latest/cypress/integration/pages/articles/testsForCanonicalOnly.js#L81) should help until the fixtures are standardised.

@jamesdonoh I'll pick this up as you're away. 👍

**Review by simonsinclair — DISMISSED** [review]

File: cypress/integration/pages/articles/tests.js (hunk 1 above)
Comment (simonsinclair) [reply cypress/integration/pages/articles/tests.js:197]: I've pushed some changes, which I tested on `live` `persian` and they seem to be doing the trick. Let's see what CI thinks.

persian

**Comment by jamesdonoh** [comment]
~This is still failing on CI apparently with the Chromatic errors I mentioned on Slack, e.g.~
```
chromatic info Uploaded your build, verifying 
[snip]
chromatic info 52/52 snapshots remain to test. (0 changes, 0 errors) 
chromatic info 51/52 snapshots remain to test. (0 changes, 0 errors) 
chromatic info 50/52 snapshots remain to test. (0 changes, 0 errors) 
chromatic info 49/52 snapshots remain to test. (1 change, 0 errors) 
chromatic info 48/52 snapshots remain to test. (2 changes, 0 errors) 
chromatic info 47/52 snapshots remain to test. (2 changes, 0 errors) 
Sending interrupt signal to process
Terminated
Makefile:10: recipe for target 'developmentTests' failed
make: *** [developmentTests] Terminated
script returned exit code 143
````

**Update**: it's nothing to do with the above as Drew pointed out, red herring. The e2e tests were failing on services like Persian, Serbian, etc. because we weren't checking the media block exists before trying to look for a caption. Fixed in c3c3ea9

**Review by dr3 — APPROVED** [review]

**Pull request merged**